lib/services/elasticsearch.js

```diff
@@ -242,15 +253,28 @@ class ElasticSearch extends Service {
 
     debug('Search: %j', esRequest);
 
+    if (scroll) {
+      const ongoingScrolls = await this._kuzzle.cacheEngine.internal.keys(
```

Comment:
Never use KEYS: it blocks other commands during its execution and can be harmful. Use scan instead.
(see https://redis.io/commands/keys)